### DISCUSSION TITLE
Want to Translate?

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#1052] [Translation] [answered]
## May 2 2021 update:
We updated our ["how to translate" page](https://scratchaddons.com/docs/localization/joining-the-localization-team/).
Follow the steps mentioned there to get invited for translation.
If you have any questions, feel free to create a discussion here on GitHub.

**Comment by Thomas282-coder** [comment]
Hello,
I want to translate scratch addons to Turkish
My Transifex name: **Translator20**
My Scratch Account Name: **Thomas282**

**Reply by Thomas282-coder** [reply]
Turkish
Translator20

**Reply by z-anonymouz** [reply]
You do Turkish? Wow! I know a little Turkish and a little German. Not enough to translate. I do know enough Arabic.

**Reply by ghost** [reply]
i wanted to do that cause im turkish but I don't have an transifex account

**Reply by z-anonymouz** [reply]
@iltersinavv You can create one!

**Reply by TheBugCoder** [reply]
Hello I want to translate to Dutch.
Scratch Name: TheBugCoder
Transifex Name: TheBugCoder

**Comment by gohoski** [comment]
Hello, I want to translate Scratch Addons to Russian. 
My Transifex username is **GOHOSKIscratch**.

**Comment by gisgreatWasTaken** [comment]


**Reply by gisgreatWasTaken** [reply]
i just realised that you aren't accepting Chinese therefore this thread is closed lol

**Reply by WorldLanguages** [reply]
Hmm, why aren't we accepting chinese?

**Reply by gohoski** [reply]
> i just realised that you aren't accepting Chinese therefore this thread is closed lol

well, they accept Chinese actually.

**Reply by GrahamSH-LLK** [reply]
@gisgreatWasTaken You can apply for chinese!

**Reply by retronbv** [reply]
wasnt chinese already done?

**Reply by z-anonymouz** [reply]
Don't think it is done.

**Comment by thegAmechanger80** [comment]
Hello.
I want to translate Scratch Addons to French.
My Transifex account name: WaRaider
My Scratch Account name: thegAmechanger80

**Reply by GrahamSH-LLK** [reply]
I've added you as our first translator to French! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/

**Comment by superscratch444tre** [comment]
Hi! I want to translate Scratch Addons to Romanian.
My Transifex account username is **superscratch444tre**
My Scratch account username is **superscratch444tre**

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to Romanian! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Comment by Minemaster552** [comment]
Hello i want tro translate scratch addons to czech!
Transifex: minemaster552

**Reply by GrahamSH-LLK** [reply]
Hi, it seems like your account is private. In order to translate, please make your account public in Transifex settings! Thank you!

**Comment by Minemaster552** [comment]
Hi!
Thanks for that, my accounts public now, but may i correct you, i said
czech not romanian!

Dne st 16. 12. 2020 20:30 uživatel GrahamSH <notifications@github.com>
napsal:

**Reply by GrahamSH-LLK** [reply]
I was responding to superscratch444tre, but it isn't clear in the email threads.
I've added you as a translator to Czech! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Comment by moeenio** [comment]
Hi! I'd like to translate to French, it's my native language. My username on Transifex is locness3.

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to French! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Comment by ghost** [comment]
I'd like to translate it to Polish! My username on transifex is robloaks.

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to Polish! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Comment by z-anonymouz** [comment]
I would love to translate it into Arabic. My username on scratch is anoymousgamerhacker and the same with for transifex.

**Reply by GrahamSH-LLK** [reply]
@anoymousgamerhacker Please make your account public. I can't find it anywhere.

You can follow the instructions at https://docs.transifex.com/account/managing-public-profile!

**Reply by z-anonymouz** [reply]
https://www.transifex.com/user/profile/anoymousgamerhacker/

I made it public now. I think there was a problem with saving.

**Reply by z-anonymouz** [reply]
@GrahamSH-LLK  Also I forgot to mention that I am very busy.  I will try to translate as much as I can.

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to Arabic! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Comment by Selman3447** [comment]
Language : Turkish
Transifex Username : Maxtorices
Transifex User Link : https://www.transifex.com/user/profile/Maxtorices/
Scratch Username : Selman3447

**Comment by Albraaxes** [comment]
FORGET IT...SOMEBODY ELSE DID IT

**Reply by z-anonymouz** [reply]
2 people can do it. Also, I took it. :)  You can still translate. Sign up and start here:https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/ 
Also, it is harder than you think. For example, since there is no such a thing as onion skinning for animation in Arabic, I just put (in Arabic)  "Onion Skinning (A technique to animate)".

**Comment by zafersafa007** [comment]
Hello there
I also want to join the translation team.
I will translate it into Turkish.
Scratch name: zafersafa007

**Reply by Secret-chest** [reply]
You need transifex name

**Comment by sfederici** [comment]
Italian
s_federici

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to Italian! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Reply by sfederici** [reply]
Does Italian have a reviewer? If not, can I be a reviewer so that all translations are final? If you need references, I'm the Italian translator of Scratch, the Scratch website, ScratchJr, the ScratchJr website, etc.

**Reply by sfederici** [reply]
If you can't make me a reviewer, be aware that the Italian translation has been completed and reviewed.

**Reply by GrahamSH-LLK** [reply]
Sorry about that! I'll make you a reviewer right now.

**Reply by sfederici** [reply]
Excellent!

**Comment by Secret-chest** [comment]
Hi! I also want to help translate Scratch Addons to Romanian.
My Transifex account username is mumu245.
My Scratch account username is also mumu245.

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to Romanian! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Comment by zafersafa007** [comment]
Me transfilex name: zafersafa07

29 Ara 2020 Sal 09:05 tarihinde Secret-chest <notifications@github.com>
şunu yazdı:

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to Turkish! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Reply by z-anonymouz** [reply]
Merhaba! (I know English and Arabic, with a tiny bit of Turkish)

**Comment by Plush-pixel** [comment]
I would like to translate to Russian
Transfix name: PlushLoler

**Comment by mxmou** [comment]
I want to translate to Slovenian (although I'm probably the only person that would use the translation).
Transifex username: mxmou

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to Slovenian! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Comment by gazalied** [comment]
Hi! I'm [@Rendangbike2](https://scratch.mit.edu/users/Rendangbike2) and I want to translate to Indonesian. There are over 100 Indonesian Scratchers in Scratch so it would be nice if they could enjoy the addons too!

**Reply by GrahamSH-LLK** [reply]
@Rendangbike2 Please make your account public, or create an account. I can't find it anywhere.

You can follow the instructions at https://docs.transifex.com/account/managing-public-profile!

**Reply by WorldLanguages** [reply]
@Rendangbike In case you haven't seen, you need to create a Transifex account to help

**Reply by gazalied** [reply]
> @Rendangbike2 Please make your account public, or create an account. I can't find it anywhere.
> 
> You can follow the instructions at https://docs.transifex.com/account/managing-public-profile!

[This? ](https://www.transifex.com/user/profile/Rendangbike)

**Reply by gazalied** [reply]
> > @Rendangbike2 Please make your account public, or create an account. I can't find it anywhere.
> > You can follow the instructions at https://docs.transifex.com/account/managing-public-profile!
> 
> [This? ](https://www.transifex.com/user/profile/Rendangbike)

This is my transifex profile

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to Indonesian! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Reply by gazalied** [reply]
How do you translate? There isn't any text in english

**Comment by hamsterrooo** [comment]
I want to translate scratch addons to Polish
My Transifex name: Jamnik_Klembus
My Scratch Account: Jamnik-Klembus

**Reply by GrahamSH-LLK** [reply]
@Jamnik-Klembus  Please make your account public. I can't find it anywhere.

You can follow the instructions at https://docs.transifex.com/account/managing-public-profile!

**Comment by Wilamaxin** [comment]
Hi,
I want to translate scratch addons to Polish
My Transifex name: Wilamaxin
My Scratch Account Name: Wilamaxin

**Reply by GrahamSH-LLK** [reply]
Hi! We've added you as a translator to Polish! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Comment by robloxboy24** [comment]
> If you want to translate Scratch Addons, please comment these below:
> 
> * Language to translate to
> * Transifex username
> 
> # Not accepting
> **Spanish** and **Japanese** are already translated, so we are not accepting new translators for those languages.

Username: Robloxboy24 OnScratch
Translate languages: Japanese

**Reply by robloxboy24** [reply]
I can also translate chinese

**Comment by m0nss** [comment]
Hello ! I would like to translate in French :) My Transifex name is ScratchCat

**Reply by GrahamSH-LLK** [reply]
@m0nss  Please make your account public. I can't find it anywhere.

You can follow the instructions at https://docs.transifex.com/account/managing-public-profile!

**Comment by SparkScratch-P** [comment]
I want to translate Scratch Addons to any of Hindi and Bangali, I am fluent in both languages. 
Transifex username: SparkScratch_P

**Reply by SparkScratch-P** [reply]
can i translate pls

**Reply by SparkScratch-P** [reply]
Transifex: SparkScratch_P.
I wanna translate to bengali and hindi

**Comment by CyberNat2000** [comment]
I'd like to translate it to Polish because I'm a Pole. My username on transifex is klasa4B (as with scratch)

**Reply by GrahamSH-LLK** [reply]
@CyberNat2000  Please make your account public. I can't find it anywhere.

You can follow the instructions at https://docs.transifex.com/account/managing-public-profile!

**Reply by CyberNat2000** [reply]
done

**Reply by GrahamSH-LLK** [reply]
Hi! We've added you as a translator to Polish! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Comment by GrahamSH-LLK** [comment]
I'll get to these requests later, please hold on.

**Comment by GGORG0** [comment]
I want to translate it to Polish, too!
Username: GGORG

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to Polish! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Comment by rens2** [comment]
I can translate to dutch
my transifex username is rens2

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to Dutch! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Comment by maDU59** [comment]
I want translate scratch addons in french, my transfex username is maDU59_ (It's the than my scratch account)

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to French! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Comment by m0nss** [comment]
Hello, my account is now in public ! You can find me

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to French! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Comment by m0nss** [comment]
Hello, thanks you ! I already did a lot of translations :)

**Reply by GrahamSH-LLK** [reply]
Awesome!

**Comment by VFDan** [comment]
Albanian, my username is VFDan

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to Albanian! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Comment by ghost** [comment]
Hi ! I'm Remixsuper_on_scratch on transifex ! I can translate scratchaddons in french !!

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to French! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Comment by m0nss** [comment]
You need to give your Transifex account name

Le mar. 12 janv. 2021 à 07:42, SparkScratch_P <notifications@github.com> a
écrit :

**Comment by scratchusernamemrtbts** [comment]
hi, I want to help translate scratch addons to thai
language: thai
transifex account username: mrtbts
i am thai people

**Reply by scratchusernamemrtbts** [reply]
my account is now public

**Reply by GrahamSH-LLK** [reply]
@scratchusernamemrtbts  Please make your account public. I can't find it anywhere.

You can follow the instructions at https://docs.transifex.com/account/managing-public-profile!

**Reply by scratchusernamemrtbts** [reply]
I already make it public

**Reply by scratchusernamemrtbts** [reply]
Can you find it again

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to Thai! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Reply by scratchusernamemrtbts** [reply]
thank you

**Comment by anatolefr** [comment]
I would like to help with the translation into my mother tongue which is French, I saw that there were already French people helping but if I can?
On transifex I'm Arqwa like on Scratch...
Thanks :)

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to French! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Comment by m0nss** [comment]
(Je suis français moi aussi) ne t'inquiète pas s'il y a déjà des français
sur la langue, car de nouveaux mots à traduire apparaissent souvent et plus
il y a de traducteurs, au mieux la qualité de la traduction est bonne, car
on peut se vérifier entre nous etc ^^

**Reply by anatolefr** [reply]
Okay, thanks but I think it's better to speak in English here, you can also reply without recreating a new message.

**Reply by z-anonymouz** [reply]
(I'm French too) don't worry if there are already French people
on the language, because new words to translate often appear and more
there are translators, at best the quality of the translation is good, because
we can check each other out etc ^^

**Comment by m0nss** [comment]
Yeah I know

**Reply by anatolefr** [reply]
If you know, so click on "Write a reply".

**Comment by cobaltt7** [comment]
I think we should have a second transifex project for the website

**Reply by GrahamSH-LLK** [reply]
A. _Not here_, make another issue.
B. First, we'd have to _make the site translateable._ We're working on it, okay? Don't just say, "Do this", provide some help, or actual ideas.

**Comment by z-anonymouz** [comment]
Can anyone take my spot as the Arabic translater? I haven't been online much and I might just quit. Thanks!

**Comment by GignacPL** [comment]
> If you want to translate Scratch Addons, please comment these below:
> 
> * Language to translate to
> * Transifex username
> 
> Alternatively, you can request to join our Transifex team at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.
> 
> # Not accepting
> **Spanish** and **Japanese** are already translated, so we are not accepting new translators for those languages.

Hi, i want to translate Scratch Addons to polish language. On Transifex my username is GignacKasztanski, and on Scratch is IgnacyIwoKarnecki.

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to Polish! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Reply by GignacPL** [reply]
Really thx!

**Comment by ghost** [comment]
The romanian translations are really messed up, so:
- Romanian
- I don't have transifex since i'm 13

**Reply by mxmou** [reply]
Unfortunately you can't translate without a Transifex account.

**Reply by ghost** [reply]
Oof

**Comment by hatrexpro** [comment]
language i want to translate : polish
my transifex account: hatrexpro

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to Polish! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Comment by Tofixrs** [comment]
language i want to translate : polish
my transifex account: Tofix26

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to Polish! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.

**Comment by GrahamSH-LLK** [comment]
I'll get to these later, so hold on.

**Comment by ghost** [comment]
Language i want to translate : Hindi
My transifex account: [Barthdry](https://www.transifex.com/user/profile/Barthdry/)

**Reply by GrahamSH-LLK** [reply]
Hold on, I'm pretty sure Hindi is denied as Scratch doesn't have it.

**Reply by scratchusernamemrtbts** [reply]
Try tell someone to translate scratch to Hindi first do you can translate SA

**Reply by ghost** [reply]
Ok then i'll switch to arabic

**Comment by ghost** [comment]
Guys are you responding our the entry is over for now

**Reply by GrahamSH-LLK** [reply]
This isn't over, but this isn't exactly a full time job. I'm a middle schooler doing this for fun

**Reply by ghost** [reply]
I know it is not full time job. Also me too

**Comment by ghost** [comment]
Okay i am switching my choice to Arabic
Language i want to translate : Arabic
My transifex account: [Barthdry](https://www.transifex.com/user/profile/Barthdry/)

**Reply by ghost** [reply]
Also now i have requested on transifex.

**Comment by ghost** [comment]
This is quite unresponsive

**Reply by GrahamSH-LLK** [reply]
Hi! I've added you as a translator to Arabic! You can start translating at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/. Sorry again for the wait!

**Reply by ghost** [reply]
Thanks for your reply.

**Reply by ghost** [reply]
Thanks for your reply

**Reply by ghost** [reply]
Thanks for your reply

**Reply by ghost** [reply]
Thanks for your reply

**Reply by GrahamSH-LLK** [reply]
Why 4 times?!

**Comment by ghost** [comment]
Sorry But the reply got repeated 4 times due to some github problem in that reply

**Comment by Albraaxes** [comment]


**Reply by GrahamSH-LLK** [reply]
Sorry, Arabic is paused until we properly support RTL languages.

**Comment by ghost** [comment]
I'd like to translate Vietnamese
Transifex username : dexeeey1

**Comment by yac3q** [comment]
I want translate to Polish. On Scratch: Janinainfa, on transifex: Janinainfa

**Comment by omeriko436** [comment]
I want to translate Turkish
Transifex and scratch username : omeriko436

**Comment by DevCascade** [comment]
Language to translate to:
Korean
Scratch username:
Developer_diary
Transifex username:
DevCascade

**Comment by TheNumber100IsTaken** [comment]
Hi! I wanna translate Norwegian Bokmål.
Scratch username: ThisFeelsScratchy
Transfiex username: TheNumber100IsTaken

**Comment by ghost** [comment]
> If you want to translate Scratch Addons, please comment these below:
> 
> * Language to translate to
> * Transifex username
> 
> Alternatively, you can request to join our Transifex team at https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/.
> 
> # Not accepting
> **Spanish** and **Japanese** are already translated, so we are not accepting new translators for those languages.

i want to do Turkish i logged in to transifex with github its iltersinav2b

**Comment by 22yeetz22** [comment]
Can I do chinese.
Usrname for transifex: 22yeetz22

**Comment by GrahamSH-LLK** [comment]
I have a pretty big backlog of translators, so I'll try to get up to date on these soon. Sorry for the wait.

**Comment by ghost** [comment]
I want to translate to Russian.
Transifex username: Sly_Little_Fox

**Comment by M-A-Z-2005** [comment]
I want to translate Scratch Addons to Russian.
My Transifex account name: **zuenko.michail**
My Scratch Account name: **M-A-Z-2005**

**Comment by m0nss** [comment]
Hello ! I would like to translate to French !

My Transifex name is ScratchCat

**Comment by WorldLanguages** [comment, marked as answer]
We updated our ["how to translate" page](https://scratchaddons.com/docs/localization/joining-the-localization-team/).
Follow the steps mentioned there to get invited for translation.
If you have any questions, feel free to create a discussion here on GitHub.
